lib/main.gi

```diff
@@ -2713,3 +2713,32 @@ end);
 ##
 #E
 ##
+
+IsMaximal:=function(N,A,D)
+The function IsMaximal checks whether A/D is maximal abelian subgroup of N/D 
+
+----------------------------
+
+IsMaximal:=function(N,A,D)
+local X0, X, Y0, Y, Z, x, W, y;

```

Comment:
General comment: in GAP, you can give more self-explanatory names to variables. For example, using `RTND` and `RTAD` instead of `X0` and `Y0` could make the latter code more understandable.